engine/src/main/resources/assets/skins/redTextColor.skin

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I'd create a skin family than an entire new skin. You can have a look at [onlinePlayersOverlay.ui](https://github.com/MovingBlocks/Terasology/blob/90283c37346787518d787df0825d28a8ff326aed/engine/src/main/resources/assets/ui/onlinePlayersOverlay.ui#L24) and the main menu skin file for an example.